datahub-frontend/play.gradle

```diff
@@ -13,9 +13,14 @@ configurations {
 }
 
 dependencies {
-  assets project(path: ':datahub-web', configuration: 'assets')
+  // assets project(path: ':datahub-web', configuration: 'assets')
```

Comment:
intentional commenting out assets?

Reply:
Oops -- this was to speed up my local build. It should _not_ be in here. Great catch!